Applications/Games/Earth Eternal - Valkal's Shadow/Online/script.json

```diff
@@ -0,0 +1,7 @@
+{
+	"scriptName": "Online",
+	"compatibleOperatingSystems": ["LINUX"],
+	"testingOperatingSystems": [],
+	"free": false,
```

Comment:
I know I'm a little late but shouldn't this be `true`?

Reply:
Oh, yes it should. It is most definitely free. Do I need to do another pull request to correct this? 

plata - yes this is my first for POL 5. I made a script for the same game when it was active for classic PlayOnLinux, but this was definitely a lot easier. I've got to say, I'm loving the new approach taken and it's nice to see JavaFX being used too.

Reply:
It's best to fix this by opening a new PR.
It's nice to hear other people like the new PlayOnLinux application :)

Reply:
Oh. Must have missed that. Anyways, it's not a big deal and can be fixed in a small pull request like @madoar suggested.

Glad that you like the new approach to writing scripts.